tests/components/airzone/test_init.py

```diff
@@ -0,0 +1,29 @@
+"""Define tests for the Airzone init."""
+
+from homeassistant.components.airzone.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+
+from .util import CONFIG, airzone_requests_mock
+
+from tests.common import MockConfigEntry
+from tests.test_util.aiohttp import mock_aiohttp_client
+
+
+async def test_unload_entry(hass):
+    """Test unload."""
+
+    with mock_aiohttp_client() as _m:
```

Comment:
https://github.com/home-assistant/core/pull/67666/files#r821451935

Reply:
Changed in 9e655b2693ded13aa0e4fa3de387da24e2c96cac